oak_runtime/src/lib.rs

```diff
@@ -1301,28 +1319,21 @@ impl Runtime {
 
         let new_node_proxy = self.clone().proxy_for_new_node(node_name);
         let new_node_id = new_node_proxy.node_id;
-        let new_node_name = new_node_proxy.node_name.clone();
 
-        self.node_configure_instance(
-            new_node_id,
-            node_type,
-            new_node_name.clone(),
-            label,
-            &node_privilege,
-        );
+        self.node_configure_instance(new_node_id, node_type, node_name, label, &node_privilege);
         let initial_handle = new_node_proxy
             .runtime
             .new_abi_handle(new_node_proxy.node_id, reader);
 
         info!(
             "{:?}: start node instance {:?} of type {} with privilege {:?}",
-            self.get_node_name(node_id),
-            new_node_name,
+            self.get_node_debug_id(node_id),
+            NodeInfo::construct_debug_id(node_name, new_node_id),
```

Comment:
It does not seem great that this logic is replicating the actual logic that assigns the debug id to the actual channel object. Is it not possible to get the debug id from the newly created channel?

Reply:
In this case yes. :) I wasn't quite sure if this log was after the node was already registered, but it is. Updated to use the `self.get_node_debug_id(new_node_id)` method.